recipe/meta.yaml

```diff
@@ -22,45 +22,53 @@ requirements:
     - cmake
     - make
   host:
-    - root >=6.16.0
+    - root_base
+    - root-binaries
     - python
-    - ipython
     - gsl
   run:
-    - python 
-    - root 
+    - python
     - future
     - ipython
-    - gsl
+    - configparser  # [py2k]
+    - scipy
+    - numpy
```

Comment:
Do you use the numpy C API? Or is numpy only used from Python?

Reply:
No, it's used from Python. Should we not specify it as a separate dependence?

Reply:
No in that case this is perfect. If you were using the C API you'd need [to do it this way](https://conda-forge.org/docs/maintainer/knowledge_base.html#linking-numpy) to account for the ABI stability.